pkg/server/warcserver/resourcehandler.go

```diff
@@ -31,15 +32,22 @@ import (
 	"github.com/nlnwa/gowarc/warcwriter"
 	"github.com/nlnwa/gowarcserver/pkg/index"
 	"github.com/nlnwa/gowarcserver/pkg/loader"
+	"github.com/nlnwa/gowarcserver/pkg/server/localhttp"
 )
 
 type resourceHandler struct {
-	loader *loader.Loader
-	db     *index.DB
+	loader   *loader.Loader
+	db       *index.DB
+	children *localhttp.Children
 }
 
 func (h *resourceHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	var renderFunc RenderFunc = func(w http.ResponseWriter, record *cdx.Cdx, cdxApi *cdxServerApi) error {
+	localhttp.FirstQuery(h, w, r, time.Second*3)
+}
+
+func (h *resourceHandler) ServeLocalHTTP(wg *sync.WaitGroup, r *http.Request) (*localhttp.Writer, error) {
```

Comment:
Unused parameter `wg *sync.WaitGroup`?

Reply:
It's used to conform to the [LocalHandler](https://github.com/nlnwa/gowarcserver/pull/24/files#diff-6002aa2ff0dafdb74b46e888c884af7de414ade2c48f2df7710f9b79bbe6ea44) interface

Reply:
Woops, your right!

Reply:
Resolved in commit ab8c719